### ISSUE TITLE
Save and continue, adding customers

### ISSUE THREAD

**Issue by normanu** [salesagility/SuiteCRM#3238]
When adding contacts on the account page, there is a button save and continue.
This button does nothing, so only saving the customer does something.

#### Expected Behavior
I expect it to add the contact and continue to a new field, so I can add more contacts.

#### Actual Behavior
Nothing happens.

#### Context
Low priority

#### Your Environment
* SuiteCRM Version used: Versie 7.8.2 Sugar / Versie 6.5.24 (Maken 509)
* Browser name and version: Chrome Version 56.0.2924.87 (64-bit)
* Environment name and version: MariaDB 10.1.21-r0, php5-5.6.30-r0
* Operating System and version: Alpine Linux 3.5.2

**Comment by kdjmonaghan** [comment]
Hello,

Perhaps I'm misunderstanding your issue (or perhaps I'm looking at the wrong page) but doing the following steps:

* Click on contacts module
* Click on the pencil next to a contact
* Change a few details and press save and continue

I get taken to the next contact in the list, which is how the functionality was structured.

From what you're saying though, you're getting this button when you're creating a contact... Can you please explain your steps on how you're doing this? Due to the functionality of what I described above, I believe this might be a bug and that may be why the button does nothing for you!

I'll look into this if you can give me steps on how you're doing this as when I go into the Contacts module and create a new Contact from the sidebar on the left, the Save and Continue button isn't there (which is what I'd expect, but, you seem to be getting different results!)

Thanks!

**Comment by normanu** [comment]
Ah if it is only for editing contacts, then yes that is different.
I do the following,

- Click on Accounts module
- Go down to contacts and click on new 
- Create a new contact and click on Save and continue

And was hoping it would save the contact and give me a new blank screen to add a new contact.

**Comment by kdjmonaghan** [comment]
Hmm, it seems like the "Save and Continue" button disappears for me not long after the page loads when I go to create a new contact. I take it, that the button remains for you?

Screenshot process of what I'm doing just to make sure I've replicated you to the T:

Select Accounts Module:

![accountsmodule](https://cloud.githubusercontent.com/assets/3587120/23706144/e9a101b4-0404-11e7-9ccf-92b87980631a.png)

Picking create new contact:

![picknewcontact](https://cloud.githubusercontent.com/assets/3587120/23706193/141ec4e4-0405-11e7-9327-3cc25b67656d.png)

No button displayed:

![nobutton](https://cloud.githubusercontent.com/assets/3587120/23706210/2a9a93b0-0405-11e7-82a6-8580e1a3a801.png)

**Comment by kdjmonaghan** [comment]
Ah never mind, I've found what you've wanted thanks to @shogunpol - I'll look into it. Thanks.

**Comment by kdjmonaghan** [comment]
Hi @imtg-suitecrm,

Your change seems to hide the save and continue button as expected. Thank-you for your contribution. :)

**Comment by kdjmonaghan** [comment]
@imtg-suitecrm Opened a PR with your changes. :)

@normanu If you'd like to try the fix, feel free to apply the changes to the files in the PR on your instance and perform a repair and rebuild on your system and be sure to have deleted your cache! Let us know if it works out as expected within your instance and if it resolves your issues!

**Comment by normanu** [comment]
Yes this issue has been resolved.

**Comment by horus68** [comment]
@normanu please reopen issue as its not solved yet, not until the solution from the PR gets merged in code.
